src/processor/ValueSetProcessor.ts

```diff
@@ -14,14 +15,64 @@ export class ValueSetProcessor {
     }
   }
 
+  static extractRules(input: ProcessableValueSet, target: ExportableValueSet): void {
```

Comment:
I noticed `extractRules` is not called in the `process` function. Is that intentional, because that line is added in #30? If we plan to merge #30 first, then that makes sense, but if this PR were to end up going in first, that should be added to `process`.

Reply:
#30 will probably be merged first, but might as well have it in both.